package.json

```diff
@@ -27,32 +27,48 @@
     "Cloud Text-to-Speech API"
   ],
   "scripts": {
-    "test": "nyc mocha",
-    "lint": "eslint '**/*.js'",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test",
+    "test": "c8 mocha build/test",
+    "samples-test": "cd samples/ && npm link ../ && npm install && npm test && cd ../",
+    "system-test": "mocha build/system-test",
     "docs": "jsdoc -c .jsdoc.js",
-    "fix": "eslint --fix '**/*.js'",
+    "lint": "gts fix && eslint --fix samples/*.js samples/**/*.js",
+    "fix": "gts fix",
     "docs-test": "linkinator docs",
-    "predocs-test": "npm run docs"
+    "check": "gts check",
+    "clean": "gts clean",
+    "compile": "tsc -p . && cp -r protos build/",
+    "compile-protos": "compileProtos src",
+    "predocs-test": "npm run docs",
+    "prepare": "npm run compile"
   },
   "dependencies": {
-    "google-gax": "^1.7.5"
+    "google-gax": "^1.9.0"
   },
   "devDependencies": {
+    "@types/mocha": "^5.2.5",
+    "@types/mv": "^2.1.0",
```

Comment:
We can drop the types for mv, ncp, and tmp

Reply:
my bad - doing it now